worlds/mmx/Client.py

```diff
@@ -0,0 +1,1124 @@
+def cmd_heal(self, amount: str = ""):
+    """
+    Request healing from EnergyLink.
+    """
+    if self.ctx.game != "Mega Man X":
+        logger.warning("This command can only be used while playing Mega Man X")
+    if (not self.ctx.server) or self.ctx.server.socket.closed or not self.ctx.client_handler.game_state:
+        logger.info(f"Must be connected to server and in game.")
+    else:
+        if self.ctx.client_handler.heal_request_command is not None:
+            logger.info(f"You already placed a healing request.")
+            return
+        if amount:
+            try:
+                amount = int(amount)
+            except:
+                logger.info(f"You need to specify how much HP you will recover.")
+                return
+            if amount <= 0:
+                logger.info(f"You need to specify how much HP you will recover.")
+                return
+            self.ctx.client_handler.heal_request_command = amount
+            logger.info(f"Requested {amount} HP from the energy pool.")
+        else:
+            logger.info(f"You need to specify how much HP you will request.")
+
+
+def cmd_refill(self, amount: str = ""):
+    """
+    Request weapon energy from EnergyLink.
+    """
+    if self.ctx.game != "Mega Man X":
+        logger.warning("This command can only be used while playing Mega Man X")
+    if (not self.ctx.server) or self.ctx.server.socket.closed or not self.ctx.client_handler.game_state:
+        logger.info(f"Must be connected to server and in game.")
+    else:
+        if self.ctx.client_handler.weapon_refill_request_command is not None:
+            logger.info(f"You already placed a weapon refill request.")
+            return
+        if amount:
+            try:
+                amount = int(amount)
+            except:
+                logger.info(f"You need to specify how much Weapon Energy you will recover.")
+                return
+            if amount <= 0:
+                logger.info(f"You need to specify how much Weapon Energy you will recover.")
+                return
+            self.ctx.client_handler.weapon_refill_request_command = amount
+            logger.info(f"Requested {amount} Weapon Energy from the energy pool.")
+        else:
+            logger.info(f"You need to specify how much Weapon Energy you will request.")
+
+def cmd_trade(self, amount: str = ""):
+    """
+    Trades HP to Weapon Energy. 1:1 ratio.
+    """
+    if self.ctx.game != "Mega Man X":
+        logger.warning("This command can only be used while playing Mega Man X")
+    if (not self.ctx.server) or self.ctx.server.socket.closed or not self.ctx.client_handler.game_state:
+        logger.info(f"Must be connected to server and in game.")
+    else:
+        if self.ctx.client_handler.trade_request is not None:
+            logger.info(f"You already placed a weapon refill request.")
+            return
+        if amount:
+            try:
+                amount = int(amount)
+            except:
```

Comment:
```suggestion
            except ValueError:
```
https://docs.astral.sh/ruff/rules/bare-except/